pkg/apis/triggers/v1alpha1/event_listener_types.go

```diff
@@ -57,11 +57,20 @@ type EventListenerSpec struct {
 	ServiceAccountName string                 `json:"serviceAccountName"`
 	Triggers           []EventListenerTrigger `json:"triggers"`
 	ServiceType        corev1.ServiceType     `json:"serviceType,omitempty"`
+	Replicas           int32                  `json:"replicas"`
```

Comment:
Should Replica be part of PodTemplate? Maybe in future have a ducktype like PodSpecable mentioned here: 
 https://github.com/tektoncd/triggers/issues/505#issuecomment-620205840
Of course, we want to give only limited set of capability even if we do ducktype.
@dibyom

Reply:
So, my understanding is that for PodTemplate or PodSpecable, the object can contain fields from a `PodSpec`: https://godoc.org/k8s.io/api/core/v1#PodSpec

`replicas` are part of the DeploymentSpec -> looking through the fields it doesn't seems like we'd want to support customizing any fields besides `replicas`. So, maybe we just leave `replicas` as part of the EL spec (maybe we can call it `deploymentReplicas` ?) What do you think @khrm ?

Reply:
I'll add this to tomorrow's WG agenda. Unless there are objections, I feel inclined to keep the current field naming!

Reply:
Thanks :) @dibyom